client/components/data/query-top-posts/README.md

```diff
@@ -0,0 +1,65 @@
+Query Top Posts
```

Comment:
Yes, that's what I was referring to. I like PRs where, as a reviewer, I don't have to spend much additional time setting anything up. I could take the example in the PR and add it somewhere to test it out. But, it would be even nicer if that was already done for me so that I can more quickly confirm things without messing around with temporary changes. :-)